tests/test_llms.py

```diff
@@ -159,6 +166,199 @@ def test_pickling(self, tmp_path: pathlib.Path) -> None:
         assert llm.router.deployment_names == rehydrated_llm.router.deployment_names
 
 
+class TestOpenAIBatchLLMModel:
+    @pytest.fixture(scope="class")
+    def config(self, request) -> dict[str, Any]:
+        model_name = request.param
+        return {
+            "model": model_name,
+            "temperature": 0.0,
+            "max_tokens": 64,
+            "batch_summary_timelimit": 24*60*60,
+            "batch_polling_interval": 5,
+        }
+
+    @pytest.mark.parametrize(
+        "config",[
+            pytest.param("gpt-4o-mini", id="chat-model"),
+            pytest.param("gpt-3.5-turbo-instruct", id="completion-model")
+        ], indirect=True
+    )
+    @pytest.mark.asyncio
+    async def test_run_prompt(self, monkeypatch, config: dict[str, Any], request) -> None:
+        ##############################
+        #                            #
+        # Create a mock batch client #
+        #                            #
+        ##############################
+
+        mock_client = AsyncMock()
+    
+        # Define mock methods for the client
+        mock_files_create = AsyncMock()
```

Comment:
```python
        mock_file_id = 'file-123'
        mock_client.files.create = AsyncMock(return_value=mock_file_id)
```

And when you refer to `mock_files_content` below, you'd change to `mock_client.files.create`. Sometimes it's easier to read the actual path, instead of temporary names like `mock_files_create`

Reply:
Thanks for the suggestion. It's indeed much cleaner